src/server/session.ts

```diff
@@ -651,16 +651,21 @@ namespace ts.server {
             }
 
             return occurrences.map(occurrence => {
-                const { fileName, isWriteAccess, textSpan } = occurrence;
+                const { fileName, isWriteAccess, textSpan, isInString } = occurrence;
                 const scriptInfo = project.getScriptInfo(fileName);
                 const start = scriptInfo.positionToLineOffset(textSpan.start);
                 const end = scriptInfo.positionToLineOffset(ts.textSpanEnd(textSpan));
-                return {
+                const result: protocol.OccurrencesResponseItem = {
                     start,
                     end,
                     file: fileName,
-                    isWriteAccess,
+                    isWriteAccess

```

Comment:
the diff only shows the changed property, and next time when someone adds/removes a property same occurs.
that happens to be the main reason why ES2017 supports trailing commas in argument lists, see https://github.com/tc39/proposal-trailing-function-commas